blueocean-pipeline-editor/src/main/js/components/editor/AgentConfiguration.jsx

```diff
@@ -142,15 +143,25 @@ export class AgentConfiguration extends Component<DefaultProps, Props, State> {
             {selectedAgent && selectedAgentMetadata && <div className="agent-parameters">
                 {selectedAgentMetadata.parameters.filter(agentConfigParamFilter(selectedAgent)).map(param => {
                     const val = this.getRealOrEmptyArg(param.name).value.value;
+                    const inputElem = (param.name === 'args' ?
```

Comment:
Might be good to move this to a function in order to support different changes that get requested, e.g. `_getAgentInputControl(param)`